web/src/containers/ActivityDetailContractorResources.js

```diff
@@ -215,11 +240,80 @@ class ContractorExpenses extends Component {
                         label={year}
                         value={contractor.years[year]}
                         onChange={this.handleYearChange(i, year)}
+                        disabled={contractor.hourly.useHourly}
```

Comment:
Kids today will never know how frickin' hard this used to be.

Reply:
😆